changelog

```diff
@@ -1,3 +1,282 @@
+Version 0.10.13
+  * Big changes:
+    * Revamped the Quarg with new artwork, ships, outfits, descriptions, and balance changes more appropriate for their tier level. (@1010todd, @Arachi-Lover, @Azure3141, @Brick63, @Quantumshark, @JimmyFiryh, @Saugia)
+    * Pausing (@TomGoodIdea, @Zarkonnen, @TheGiraffe3, @Amazinite)
+    * OpenGL frag and vert shader code is now defined in data files, instead of within the compiled binary, and so can be customized without recompilation. (@tibetiroka)
```

Comment:
```suggestion
    * OpenGL fragment and vertex shader code is now defined in data files, instead of within the compiled binary, and so can be customized without recompilation. (@tibetiroka)
```
They can also be customized by plugins, btw.